.github/build

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+make release
```

Comment:
note: I actually think this will fail because it invokes docker.... but I'm not sure. I'd be ok to invoke `make local-release` as long as we can guarantee the Go version that is used to build the release.

Reply:
Looking at the source code, I'm not sure what exactly the build action does, it just seems to invoke this script, nothing else: https://github.com/skx/github-action-build

Maybe we can run the build with a normal shell step into the `release/` folder? 

For sure, we probably want to test this on a throw-away repo first before merging it, to avoid spamming subscribed users with test tags.

Reply:
OK, I tested this on a fork. The artifacts generation step indeed failed but it turns out that `skx/github-action-build` does nothing special and using a simple `make release` as a step instead is enough and works just fine. I think this PR is good to go as is.